service/settings.py

```diff
@@ -264,7 +264,7 @@ def getInstance(cls):
     def __init__(self):
         serviceHTMLExportDefaultSettings = {
             "enabled": False,
-            "path"   : config.pyfaPath + os.sep + 'pyfaFits.html',
+            "path"   : os.path.join(config.savePath, 'pyfaFits.html'),
```

Comment:
This isn't fine.

Unfortunately it's a problem for paths with unicode in them.  (Any `os.path` stuff is.)  Hence the hacky way of dealing with that here.

Umm, plus as mentioned, this isn't really the right spot for these to go.